metadata-models/src/main/pegasus/com/linkedin/ermodelrelation/EditableERModelRelationProperties.pdl

```diff
@@ -0,0 +1,31 @@
+namespace com.linkedin.ermodelrelation
+
+import com.linkedin.common.ChangeAuditStamps
+
+
+/**
+ * EditableERModelRelationProperties stores editable changes made to ermodelrelation properties. This separates changes made from
+ * ingestion pipelines and edits in the UI to avoid accidental overwrites of user-provided data by ingestion pipelines
+ */
+@Aspect = {
+  "name": "editableERModelRelationProperties"
+}
+record EditableERModelRelationProperties includes ChangeAuditStamps {
```

Comment:
since for now this is a datahub entity that is generated in datahub, let's get rid of this editable properties aspect.

usually we have editable properties aspects when we want to preserve the original properties that we generate for an asset from the source system in ingestion. Since these entities are created through datahub (in the UI or the API) I don't believe we need both types of properties aspects